docs/toc.yml

```diff
@@ -147,6 +147,8 @@
         - name: Update API and manifest version
           href: develop/update-your-javascript-api-for-office-and-manifest-schema-version.md
           displayName: manifest
+      - name: App package for Microsoft 365
+        href: overview/app-package-for-microsoft-365.md
```

Comment:
Not a blocker, but consider moving this item before the manifest references since the manifest is part of the app package.